gps/app/views/onboarding/Onboarding1.js

```diff
@@ -140,16 +140,16 @@ const styles = StyleSheet.create({
   // eslint-disable-next-line react-native/no-color-literals
   languageSelector: {
     // alpha needs to be in the bg color otherwise it fades the contained text
```

Comment:
looks like this comment is no longer relevant? what do you think about removing it?